standardization/src/main/scala/za/co/absa/enceladus/standardization/CmdConfig.scala

```diff
@@ -134,6 +147,25 @@ object CmdConfig {
           success
         else
           failure("The --delimiter option is supported only for CSV raw data format"))
+
+    opt[String]("csv-quote").optional().action((value, config) =>
```

Comment:
Possibly. The value ends up as an argument to `.option(key, _)` which does not have a Char overload, just String. But it might be a good idea to have a Char anyway and convert it to String afterwards. Will check this possibility.

Reply:
It worked perfectly. Made `delimiter` to be `Char` as well.

Reply:
Sorry, it didn't work after all. Sometime it works, sometimes not. Reverting to String.

I think it does not work when UTF-8 encoding of a char is larger than 1 byte. 
```
Error: Option --delimiter failed when given '¡'. '[B@38ed139b' is not a char
```

Reply:
Just wondering where the problem is coming from. `Char` is a 16-bit unsigned integer, therefore it should have nor problem with UTF-8 🤔

Reply:
Yeah, I think it is a `scopt` specific issue.

Reply:
We could log it in `scopt` and maybe do something about it on a later date. How about that?

Reply:
https://github.com/scopt/scopt/issues/258